README.md

```diff
@@ -11,6 +11,9 @@ The following compression codings are supported:
 
   - deflate
   - gzip
+  - br (brotli)
+
+**Note** Brotli provides [better and faster compression then gzip or deflate](https://expeditedsecurity.com/blog/nginx-brotli/), but is supported only since Node.js versions v11.7.0 and v10.16.0.
```

Comment:
I don't think this makes the statement any more clearer. I would rather we just leave out editorial comments like this all together and just keep it to the facts of the module -- that br is only supported in specific modules. Even if the editorial was left in, what good does it serve the users of this module? There is no option for them to disable or enable br and the usage of it is left completely up to the clients accessing the server. Besides, this link demonstrates how to use br with a completely different module than this module, which is just going to be confusing.

Reply:
Cleared.